Classes/Command/Input/Question/AbstractAttributeQuestion.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace FriendsOfTYPO3\Kickstarter\Command\Input\Question;
+
+use FriendsOfTYPO3\Kickstarter\Context\CommandContext;
+use FriendsOfTYPO3\Kickstarter\Information\InformationInterface;
+use FriendsOfTYPO3\Kickstarter\Information\Normalization\UseNormalizer;
+use FriendsOfTYPO3\Kickstarter\Information\Validatation\UseValidator;
+use ReflectionClass;
+use Symfony\Component\Console\Helper\SymfonyQuestionHelper;
+use Symfony\Component\Console\Question\Question;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+abstract readonly class AbstractAttributeQuestion implements AttributeQuestionInterface
+{
+    abstract protected function getQuestion(): array;
+
+    public function getArgumentName(): string
+    {
+        $calledClass = static::class;
+
+        if (!defined($calledClass . '::ARGUMENT_NAME')) {
+            throw new \RuntimeException(
+                sprintf('Class %s must define the constant ARGUMENT_NAME.', $calledClass),
+                3362475211
+            );
+        }
+
+        return $calledClass::ARGUMENT_NAME;
+    }
+
+    public function getInformationClass(): string
+    {
+        $calledClass = static::class;
+
+        if (!defined($calledClass . '::INFORMATION_CLASS')) {
+            throw new \RuntimeException(
+                sprintf('Class %s must define the constant INFORMATION_CLASS.', $calledClass),
+                3362475211
```

Comment:
What is that value? IMO we use the timestamp in TYPO3, but somewhere below the value is 6481694059. This can not be any kind of timestamp. How do you prevent duplicated values?

Reply:
I am using AddErrorCodeToExceptionRector rule which applies a random number random_int(1000000000, 9999999999);

The large number of available numbers ensures the number to be unique